lbrynet/daemon/Daemon.py

```diff
@@ -2004,9 +2002,11 @@ def jsonrpc_publish(self, name, bid, metadata=None, file_path=None, fee=None, ti
         if bid <= 0.0:
             raise Exception("Invalid bid")
 
-        if bid >= self.session.wallet.get_balance():
-            raise InsufficientFundsError('Insufficient funds. ' \
-                                         'Make sure you have enough LBC to deposit')
+        amt = yield self.session.wallet.get_max_usable_balance_for_claim(name)
+        if bid > amt:
+            raise InsufficientFundsError(
+                "Please lower the bid value, the max amount you can specify for this claim is {}"
```

Comment:
In user facing messages we should spell out 'max' as maximum. If this text doesn't make it to the user then never mind.